san/batch.py

```diff
@@ -11,24 +12,36 @@ def __init__(self):
     def get(self, dataset, **kwargs):
         self.queries.append([dataset, kwargs])
 
-    def execute(self):
-        result = []
+    def __create_batched_query_string(self):
         batched_queries = []
 
-        for idx, q in enumerate(self.queries):
-            batched_queries.append(get_gql_query(idx, q[0], **q[1]))
-
-        gql_string = self.__batch_gql_queries(batched_queries)
-        res = execute_gql(gql_string)
+        for idx, query in enumerate(self.queries):
+            [metric, slug] = query[0].split('/')
+            if metric in V2_METRIC_QUERIES:
+                batched_queries.append(
+                    san.sanbase_graphql.get_metric(
+                        idx, metric, slug, **query[1]))
+            else:
+                batched_queries.append(get_gql_query(idx, query[0], **query[1]))
+        self.__batch_gql_queries(batched_queries)
+        return self.__batch_gql_queries(batched_queries)
+
+    def __transform_batch_result(self, executed_gql):
+        result = []
 
-        idxs = sorted([int(k.split('_')[1]) for k in res.keys()])
+        idxs = sorted([int(k.split('_')[1]) for k in executed_gql.keys()])
         for idx in idxs:
             query = self.queries[idx][0].split("/")[0]
-            df = transform_query_result(idx, query, res)
+            df = transform_query_result(idx, query, executed_gql)
             result.append(df)
 
         return result
 
+    def execute(self):
```

Comment:
lets move this below `get` because that is our public interface. And move helper functions starting with `__` to the bottom

Reply:
Oh, my bad, I will do it